crates/sui/tests/ptb_files_tests_bad.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::CommandFactory;
+use std::{collections::BTreeMap, path::Path};
+use sui::ptb::{ptb::PTB, ptb_parser::parser::PTBParser};
+
+// the PTB parser will complain, but clap will be fine
+fn test_bad_ptb(path: &Path) -> datatest_stable::Result<()> {
+    let ptb = PTB::default();
+    let cmd = PTB::command();
+    let file = path.to_str().unwrap();
+    let args = cmd.get_matches_from(vec!["ptb", "--file", file]);
+    let commands = ptb.from_matches(&args, None, &mut BTreeMap::new());
+    assert!(commands.is_ok());
+
+    let mut parser = PTBParser::new();
+    for command in commands.unwrap() {
+        parser.parse(command.1);
+    }
+    let (parsed, errors) = parser.finish();
+    assert!(!errors.is_empty());
```

Comment:
Can we render the errors here to? This will be good for negative tests and to make sure error results stay stable/we know when/how they're changing.

Reply:
Just rendering them? Sure thing.

Reply:
Yea, I'd love to have `.exp` files that we can render the errors into -- this should allow us to make sure errors are correct (and that changes don't change them in unexpected ways). We have some existing infrastructure around this already on the Move side, (or we could possibly use `insta` for this).